client/packages/system/src/Patient/JsonForms/components/BMI.tsx

```diff
@@ -28,7 +28,10 @@ const UIComponent = (props: ControlProps) => {
 
     if (!handleChange || !w) return;
 
-    const bmi = round(w / (h * h));
+    let bmi = round(w / (h * h));
+    if (!isFinite(bmi)) {
+      bmi = 0
```

Comment:
Nit: could you please set to undefined?

Reply:
[d04c85c](https://github.com/openmsupply/open-msupply/pull/869/commits/d04c85c2d5b78c232334b2ebd4ffd6f03c369c21)